x-pack/plugins/lens/public/indexpattern_plugin/indexpattern.test.tsx

```diff
@@ -248,11 +263,184 @@ describe('IndexPattern Data Source', () => {
         expect(wrapper).toMatchSnapshot();
       });
 
+      describe('drag and drop', () => {
+        function dragDropState() {
+          return {
+            ...state,
+            currentIndexPatternId: 'foo',
+            indexPatterns: {
+              foo: {
+                id: 'foo',
+                title: 'Foo pattern',
+                fields: [{ aggregatable: true, name: 'bar', searchable: true, type: 'number' }],
+              },
+            },
+          };
+        }
+
+        it('is not droppable if no drag is happening', () => {
+          const component = mount(
+            <IndexPatternDimensionPanel
+              dragDropContext={dragDropContext}
+              state={dragDropState()}
+              setState={() => {}}
+              columnId={'col2'}
+              filterOperations={() => true}
+            />
+          );
+
+          const dropTarget = component
+            .find('[data-test-subj="indexPattern-dropTarget"]')
+            .first()
+            .props();
+
+          expect((dropTarget as { droppable: boolean }).droppable).toBeFalsy();
```

Comment:
I think this test might be simpler as: `expect(component.find('...').first().prop('droppable')).toBeFalsy();` - especially since you reuse the same structure of the assertion in the following tests.